package-testing/android-relay/app/build.gradle.kts

```diff
@@ -0,0 +1,123 @@
+import com.android.build.gradle.internal.cxx.configure.gradleLocalProperties
+
+plugins {
+    alias(libs.plugins.android.application)
+    alias(libs.plugins.jetbrains.kotlin.android)
+    jacoco
+
+    id("com.ncorti.ktfmt.gradle") version "0.20.1"
+}
+
+apply {
+    from("./jacoco.gradle")
+}
+
+jacoco {
+    toolVersion = "0.8.12"
+    reportsDirectory = layout.buildDirectory.dir("customJacocoReportDir")
+}
+
+
+val apiKey: String = gradleLocalProperties(
+    project.rootDir,
+    providers
+).getProperty("cloud.eppo.apiKey")
+
+val testRunnerHost: String = System.getenv("TEST_RUNNER_HOST") ?: "http://10.0.2.2"
+val testRunnerPort: String = System.getenv("TEST_RUNNER_PORT")  ?: "3000"
+
+val eppoAPIHost: String = System.getenv("EPPO_API_HOST") ?: "http://10.0.2.2"
+val appoAPIPort: String = System.getenv("EPPO_API_PORT")  ?: "5000"
+
+
+android {
+    buildFeatures.buildConfig = true
+
+    namespace = "cloud.eppo.android.sdkrelay"
+    compileSdk = 34
+
+    defaultConfig {
+        applicationId = "cloud.eppo.android.sdkrelay"
+        minSdk = 26
+        targetSdk = 34
+        versionCode = 1
+        versionName = "1.0"
+
+        testInstrumentationRunner = "androidx.test.runner.AndroidJUnitRunner"
+        vectorDrawables {
+            useSupportLibrary = true
+        }
+
+        buildConfigField( "String", "API_KEY", "\"" + apiKey + "\"")
+        buildConfigField( "String", "TEST_RUNNER_HOST", "\"" +testRunnerHost+ "\"")
+        buildConfigField( "String", "TEST_RUNNER_PORT", "\"" +testRunnerPort+ "\"")
+        buildConfigField( "String", "EPPO_API_HOST", "\"" +eppoAPIHost+ "\"")
+        buildConfigField( "String", "EPPO_API_PORT", "\"" +appoAPIPort+ "\"")
+    }
+
+    buildTypes {
+        debug {
+            enableUnitTestCoverage = true
+            enableAndroidTestCoverage = true
+        }
+        release {
+            isMinifyEnabled = true
+            isShrinkResources = true
+            proguardFiles(
+                getDefaultProguardFile("proguard-android-optimize.txt"),
+                "proguard-rules.pro"
+            )
+        }
+    }
+    compileOptions {
+        sourceCompatibility = JavaVersion.VERSION_1_8
+        targetCompatibility = JavaVersion.VERSION_1_8
+    }
+    kotlinOptions {
+        jvmTarget = "1.8"
```

Comment:
Today I learned Java 8 is developer version 1.8

Reply:
It can be so confusing. I spent half an hour reading all about it when I was fighting with jitpack earlier

Reply:
yup confusing af :)